### ISSUE TITLE
[Bug]: While navigating to “Controls, Actions, Interactions, Accessibility, Performance” tab items narrator announce twice announcement for tab items.

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#24123]
### Describe the bug

While navigating to “Controls, Actions, Interactions, Accessibility, Performance” tab items narrator announce twice announcement for tab items.

Actual Result:
While navigating to “Controls, Actions, Interactions, Accessibility, Performance” tab items narrator announce twice announcement for tab items.

Expected Result:
While navigating to “Controls, Actions, Interactions, Accessibility, Performance” tab items narrator should announce only once.


### To Reproduce

1. Open URL: [Common Navigation - Scrolling Panel ⋅ Storybook (windows.net)](https://vivacommonstorybooktest.blob.core.windows.net/$web/index.html?path=/story/common-navigation--scrolling-panel&globals=backgrounds.grid:!false;backgrounds.value:transparent)
2. Navigate to “App bar” button at right side of the pane invoke it and navigate to “Default” link invoke it. 
3. Now navigate to “Controls, Actions, Interactions, Accessibility, Performance” tab items and observe the narrator announcement           


Attachments:

https://github.com/storybookjs/storybook/assets/93735775/930a7a65-3586-4439-848e-99ed81204bf5

                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     

### System

```bash
OS: Windows 11
OS version: 22H2 (OS Build 22621.2134)
Edge Version: 116.0.1938.69 (Official build) (64-bit)
```


### Additional context

_No response_

**Comment by supriya-2108** [comment]
Hello team please assign this issue to me

**Comment by thinkcolallen** [comment]
Hello may I have this fix? Thanks

**Comment by wchon** [comment]
Hi may I have this fix? Thanks

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by Sidnioulz** [comment]
This specific one could be a narrator bug. We'll need to address #32235 first to find out.

**Comment by msftedad** [comment]
We have closed this issue at our end. Closing this for now.

**Comment by Sidnioulz** [comment]
@msftedad thank you for the update!
